index-import.cpp

```diff
@@ -237,6 +277,21 @@ remapUnit(StringRef outputUnitsPath, StringRef inputUnitPath,
       break;
     }
     case IndexUnitReader::DependencyKind::Record:
+      if (cloneDepRecords) {
```

Comment:
There is a lager subthread on culling deps for remotely cached unit files. In my testing Xcode loading them just fine even if the records are gone, and the program `validate-index` checks out.